eis_qgis_plugin/wizard/eda/plots/parallel_coordinates.py

```diff
@@ -112,6 +141,31 @@ def prepare_data(self):
         data, y_min, y_max = self._normalize_data(data)
 
         return data, fields, y_min, y_max
+    
+
+    def prepare_raster_data(self):
+        # Get input values
+        rasters = self.raster_table.get_layers()
+        raster_names = [raster.name() for raster in rasters]
+
+        height = rasters[0].height()
+        width = rasters[0].width()
+
+        # Check that all raster dimensions match
+        for raster in rasters[1:]:
+            height_n = raster.height()
+            width_n = raster.width()
+            if height_n != height or width_n != width:
+                raise ValueError("All rasters must have the same dimensions.")
```

Comment:
You should check that the raster grids match otherwise too, dimension check alone is not enough. Basically the same check as "check raster grids" EIS toolkit does